azure/v11.4.0/release.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: v11.4.0
```

Comment:
To reduce chance of migration steps being skipped, next Azure platform release with upgraded Kubernetes and Calico should be 11.5.0 even when 11.5.0 remains on same Kubernetes minor version.

Ideally there should be no 11.4.* patch releases. In case any Azure 11.4.* release does get published it's release notes should include or reference the 11.4.0 migration processes steps.